### PR TITLE
Weathers

### PR DESCRIPTION
**Pull request by Jackriip**
## Changelog
:cl:
add: Added a better weather system.
/:cl:

### CODE REVIEW

File: code/datums/weather/weather_types/hailstorm.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	affects_underground = FALSE
+	thunder_chance = 4
+
+	sound_active_outside = /datum/looping_sound/weather/wind/indoors
+	sound_active_inside = /datum/looping_sound/weather/wind
+	sound_weak_outside = /datum/looping_sound/weather/wind/indoors
+	sound_weak_inside = /datum/looping_sound/weather/wind
+
+/datum/weather/hailstorm/weather_act(mob/living/living_mob)
+	/// Think of some good solution of how weather should affect monsters and how they should be resistant to things like this
```
Comment (SuperNovaa41) [inline code/datums/weather/weather_types/hailstorm.dm]: this shouldn't be an autodoc comment 
```suggestion
	// Think of some good solution of how weather should affect monsters and how they should be resistant to things like this
```

File: code/datums/weather/weather_types/radiation_storm.dm [hunk 2]
```diff
@@ -48,7 +48,7 @@
 	priority_announce("The radiation threat has passed. Please return to your workplaces.", "Anomaly Alert")
 	status_alarm(FALSE)
 
-/datum/weather/rad_storm/proc/status_alarm(active)	//Makes the status displays show the radiation warning for those who missed the announcement.
+/datum/weather/rad_storm/proc/status_alarm(active) //Makes the status displays show the radiation warning for those who missed the announcement.
```
Comment (SuperNovaa41) [inline code/datums/weather/weather_types/radiation_storm.dm]: this SHOULD be an autodoc comment
```suggestion
///Makes the status displays show the radiation warning for those who missed the announcement.
/datum/weather/rad_storm/proc/status_alarm(active)
```

File: code/datums/weather/weather_types/snow_storm.dm [hunk 3]
```diff
@@ -2,26 +2,32 @@
...
 
 	weather_message = "<span class='userdanger'><i>Harsh winds pick up as dense snow begins to fall from the sky! Seek shelter!</i></span>"
 	weather_overlay = "snow_storm"
 	weather_duration_lower = 600
 	weather_duration_upper = 1500
 
 	end_duration = 100
-	end_message = "<span class='boldannounce'>The snowfall dies down, it should be safe to go outside again.</span>"
+	end_overlay = "snowfall_light"
+	end_message = "<span class='notice'>The snowfall dies down, it should be safe to go outside again.</span>"
```
Comment (SuperNovaa41) [inline code/datums/weather/weather_types/snow_storm.dm:16]:
```suggestion
	end_message = span_notice("The snowfall dies down, it should be safe to go outside again.")
```

File: code/datums/weather/weather_types/shroud_storm.dm [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+	sound_active_outside = /datum/looping_sound/weather/wind/indoors
+	sound_active_inside = /datum/looping_sound/weather/wind
+	sound_weak_outside = /datum/looping_sound/weather/wind/indoors
+	sound_weak_inside = /datum/looping_sound/weather/wind
+
+	multiply_blend_on_main_stage = TRUE
+
+/datum/weather/shroud_storm/weather_act(mob/living/living_mob)
+	/// Think of some good solution of how weather should affect monsters and how they should be resistant to things like this
```
Comment (SuperNovaa41) [inline code/datums/weather/weather_types/shroud_storm.dm]:
```suggestion
	// Think of some good solution of how weather should affect monsters and how they should be resistant to things like this
```

File: code/datums/weather/weather_types/snow_storm.dm [hunk 5]
```diff
@@ -2,26 +2,32 @@
 	name = "snow storm"
 	desc = "Harsh snowstorms roam the topside of this arctic planet, burying any area unfortunate enough to be in its path."
 
-	telegraph_message = "<span class='warning'>Drifting particles of snow begin to dust the surrounding area..</span>"
+	telegraph_message ="<span class='danger'>Drifting particles of snow begin to dust the surrounding area..</span>"
```
Comment (SuperNovaa41) [inline code/datums/weather/weather_types/snow_storm.dm:5]:
```suggestion
	telegraph_message = span_danger("Drifting particles of snow begin to dust the surrounding area...")
```

File: code/datums/weather/weather_types/shroud_storm.dm [hunk 6]
```diff
@@ -0,0 +1,41 @@
+/datum/weather/shroud_storm
+	name = "shroudstorm"
+	desc = "Zap."
+
+	telegraph_message = "<span class='danger'>You hear thunder in the distance, static electricity rising in the air, wind starts to pickup..</span>"
```
Comment (SuperNovaa41) [inline code/datums/weather/weather_types/shroud_storm.dm:5]: Likewise for all of these really, even if you're copy-pasting this code, it should be updated for consistency

**Comment by SuperNovaa41** [comment]
What's going on with this @Jackriip

**Comment by Jackriip** [comment]
> What's going on with this @Jackriip 

I'll get it fixed up later today

**Comment by Jackriip** [comment]
@SuperNovaa41 applied your suggestions. If you think its g2g I'd like to go ahead and merge this.

**Comment by Jackriip** [comment]
@SuperNovaa41 can we merge this?

**Pull request merged**